src/DynamoCore/DSEngine/CodeCompletionServices.cs

```diff
@@ -219,6 +182,40 @@ internal IEnumerable<CompletionData> SearchTypes(string stringToComplete, Elemen
             return candidates.Select(x => CompletionData.ConvertMirrorToCompletionData(x));
         }
 
+        private void AddTypesToCompletionData(string stringToComplete, List<CompletionData> completions, ElementResolver resolver)
```

Comment:
I assume some of these calls will be called for every, keystroke/time interval ... do you observe any performance lag when you try to do a normal workflow?